### DISCUSSION TITLE
aide

### DISCUSSION THREAD

**Discussion by zami87** [community/community#30406] [New to GitHub]
bonjour
je suis retraitée le domaine multimédia est ma passion comment comprendre un peu plus le github pour expliquer mon projet
merci

**Comment by doguedogue** [comment]
Excellent, il est important d'exposer toutes les connaissances que vous avez. 🚀🚀🚀

1. Créer un nouveau projet
2. Ajoutez un fichier README.md (markdown) pour exposer toutes vos connaissances
3. Si vous avez du code, téléchargez-le

Le fichier (markdown) peut être grandement enrichi d'images, de gifs et de graphiques.

Si j'ai répondu à votre question, veuillez la marquer comme répondue.
